src/test/java/teammates/test/cases/datatransfer/InstructorPrivilegesTest.java

```diff
@@ -214,8 +214,8 @@ public void testUpdatePrivilegeInSectionLevel() {
         assertFalse(sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS));
         assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_SESSION_IN_SECTIONS));
         assertEquals(null, sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_SUBMIT_SESSION_IN_SECTIONS));
-        assertEquals(null, sectionPrivileges.get(Const.ParamsNames
-                                                     .INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
+        assertEquals(null,
+                     sectionPrivileges.get(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION_COMMENT_IN_SECTIONS));
```

Comment:
We've switched to using 8-space indentation rather than aligning the arguments, so delete 5 spaces.